spec/latest/json-ld-api/index.html

```diff
@@ -40,33 +40,48 @@
       githubAPI:              "https://api.github.com/repos/json-ld/json-ld.org",
       includePermalinks:      true,
       noRecTrack:             false,
-      testSuiteURIkey:       "http://json-ld.org/test-suite/", 
+      testSuiteURIkey:       "http://json-ld.org/test-suite/",
       postProcess:            [internalizeTermListReferences],
 
       // editors, add as many as you like
       // only "name" is required
       editors:  [
-          { name: "Markus Lanthaler", url: "http://www.markus-lanthaler.com/",
```

Comment:
When specs are updated, it's quite common that editors change. Manu's approach preserves editors and authors from 1.0 into 1.1. I believe this was the case when you edited RDF Concepts 1.1.

The fact is, even if there is not a substantial amount of change to the document, there is considerable effort put in by me to make it happen. In many cases, reducing the change is more work than changing things more.

The fact is, we all put a lot of work into 1.0, and I've put a lot of work into the 1.1 effort.

What would you say a fair way to deal with this is?